### ISSUE TITLE
openntpd wont start (as all parts using /etc/rc.subr)

### ISSUE THREAD

**Issue by tps800** [Stichting-MINIX-Research-Foundation/minix#60]
Install openntpd, configure it, then

```shell
# sh /usr/pkg/etc/rc.d/openntpd start
Usage: ps [-][aeflx]
Starting ntpd.
# ps ax | grep ntp
  449  p0  0:00 grep ntp
#
```
no process was started. This is because in /etc/rc.subr:
```
273                  ps -o "pid,command" '"$_psargs"' |
```

but "ps" doesn't know anything about an option "-o"! It
does only know about options "aeflx", with or without a leading dash.

**Comment by tps800** [comment]
/etc/rc.subr isn't pkgsrc. It's part of the base system to support writing /etc/rc.d-scripts.

**Comment by dcvmoole** [comment]
Since rc.subr relies quite heavily on the netbsd ps(1) syntax, the safest way to fix this would be to import netbsd ps(1). That is however not going to be trivial..

**Comment by dcvmoole** [comment]
NetBSD ps(1) has been merged as 97189037b1c31a92749a06453951cb811bee1eb0, thereby resolving this issue.
